### ISSUE TITLE
Highcharts package on jsDelivr is too large

### ISSUE THREAD

**Issue by evertharmeling** [highcharts/highcharts#20124]
#### Expected behaviour
A working file listing on [jsDelivr](https://www.jsdelivr.com/package/npm/highcharts?tab=files) and not receive the error `Package size exceeded the configured limit of 150 MB.`

#### Actual behaviour
Recently the `highcharts` package is increased in size, which has triggered [jsDelvr](https://www.jsdelivr.com/package/npm/highcharts?tab=files) a `Package size exceeded the configured limit of 150 MB.`. This [caused problems](https://github.com/symfony/symfony/issues/52533) for the `symfony/asset-mapper` component as it tightly integrated with jsDelivr.

There are [issues](https://github.com/jsdelivr/jsdelivr/issues?q=is%3Aissue+large+increase) that the limit could be increased, however it's probably interesting what caused the increase, as it's a fairly big difference since the initial `11.0.0` version.

```
Failed to load the file listing.
Package size exceeded the configured limit of 150 MB.

Consider [asking the author to remove unnecessary files](https://github.com/highcharts/highcharts-dist/issues/new/?title=npm%20package%20too%20big%20for%20jsDelivr&body=I%27m%20trying%20to%20load%20this%20package%20from%20[jsDelivr%20CDN](https://www.jsdelivr.com/package/npm/highcharts?tab=files)%20but%20it%20fails%20because%20the%20package%20is%20over%2050%20MB.%20Are%20there%20any%20files%20that%20could%20be%20removed%20to%20reduce%20the%20package%20size?) to make the package smaller.

You may still find the files you need in the [GitHub repository](https://github.com/highcharts/highcharts-dist/) and include them in your project like this:
https://cdn.jsdelivr.net/gh/highcharts/highcharts-dist@sharedVersion/file

Read more about [GitHub CDN](https://www.jsdelivr.com/documentation#id-github), which can load individual files even if the overall package size exceeds the limit.
```

#### Product version
11.2.0

#### Related
- https://github.com/highcharts/highcharts/issues/20116

**Comment by karolkolodziej** [comment]
Hi @evertharmeling! 
Thanks for describing everything in such detail! 

We are aware that with `v11.2.0` some unnecessary things made their way to the bundle. We are already working on it, and it certainly will be fixed before the next release!

**Comment by evertharmeling** [comment]
No problem, good to hear! Thanks!

**Comment by MantraMedia** [comment]
@evertharmeling how did you solve this? did you just go with the 11.0.0 or not use HighCharts at all with asset mapper?

**Comment by evertharmeling** [comment]
@MantraMedia yes, for now I sticked to a lower v11 version (not sure if it was 11.0.0 or 11.0.1, the most recent version that worked)
